build.gradle

```diff
@@ -51,12 +51,14 @@ repositories {
 	flatDir {
 		dirs "libs"
 	}
+
+	jcenter()
 }
 
 dependencies {
 	compile group: "net.shadowfacts", name: "ShadowMC", version: mc_version + "-" + shadowmc_version
 
-	compile group: "net.dv8tion", name: "JDA", version: "1.2.2_167", classifier: "withDependencies"
+	shade 'net.dv8tion:JDA:2.0.0_242'
```

Comment:
The `shade` configuration doesn't exist in 1.8.9. Use the same format as before, just change the version. (The JDA jar should also be renamed so the `withDependencies` classifier follows the standard Maven convention)